PowerEditor/src/Notepad_plus.cpp

```diff
@@ -174,6 +174,11 @@ Notepad_plus::Notepad_plus()
 		is_admin = false;
 
 	_isAdministrator = is_admin ? true : false;
+
+	// Enable new style save dialog.
+	if (ver >= WV_VISTA) {

```

Comment:
@SinghRajenM  No this check is for `Save Dialog` only, because the save dialog style in the image above is introduced in Windows Vista.

For the `Folder Browse Dialog` I already tested it on Windows XP and it's work perfectly, but I forgot to say that.